### ISSUE TITLE
InterfaceGenerator and #include "..\<inputFileName>.h"

### ISSUE THREAD

**Issue by TonyMSantander** [xlw/xlw#27]
Hello.

Commit c7ff826 on July 20 2020 changes the way that the generated cpp file includes the header file from which it was generated.

Say I created my addin functions in AddinFuncs.h. The generated file (say xlwAddinFuncs.cpp) now has this:
#include "..\AddinFuncs.h"

Before this commit, it had:
#include "AddinFuncs.h"

This change forces me to generate the cpp file in a sub-directory of the directory containing AddinFuncs.h. I don't think that it should make this decision; what if I want my build system to place all generated files (i.e. object files and source files) in a different location?

I should have the freedom to do that, and to configure my include directories so that the compiler finds AddinFuncs.h.

I propose that this change be reverted to the previous behaviour.

Regards

Tony

**Comment by keithalewis** [comment]
Why not checkout the commit that works for you?

**Comment by TonyMSantander** [comment]
Yes, I changed the code to the previous version. I propose that the master code revert as well because the decision as to where the generated cpp file exists in relation to the source header file should not be determined by the code; it should be determined by the build environment.

**Comment by keithalewis** [comment]
Set the include directories in your build system. You could also submit a PR.

**Comment by TonyMSantander** [comment]
I don't think that I have explained the problem well. Rather than try again, perhaps you can tell me why it's a good idea to generate the cpp file with #include "..\AddinFuncs.h" instead of #include "AddinFuncs.h".

**Comment by keithalewis** [comment]
I don't think it is a good idea to generate any code. The [library](https://github.com/xlladdins/xll) I wrote for creating add-ins doesn't do that.

**Comment by TonyMSantander** [comment]
This GitHub repository and issues manager is about XLW, and XLW generates code. Your library is interesting; I'll take a better look at it later.
